test/xcode-specs.js

```diff
@@ -83,4 +83,10 @@ describe('xcode @skip-linux', () => {
     (parseFloat(version)-6.1).should.be.at.least(0);
   });
 
+  it('should get a list of iOS devices', async() => {
+    let devices = await xcode.getConnectedDevices();
+    should.exist(devices);
```

Comment:
Don't bother for now, we'll probably need to review all those tests, I'll give it a go after appium-ci.